Sources/SwiftBasicFormat/generated/BasicFormat.swift

```diff
@@ -136,8 +136,16 @@ open class BasicFormat: SyntaxRewriter {
     }
   }
   
-  open func requiresTrailingSpace(_ tokenKind: TokenKind) -> Bool {
-    switch tokenKind {
+  open func requiresTrailingSpace(_ token: TokenSyntax) -> Bool {
+    if let nextTokenKind = token.nextToken(viewMode: .sourceAccurate)?.tokenKind {
+      switch nextTokenKind {
+      case .exclamationMark, .postfixQuestionMark: 
+        return false
+      default: 
+        break 
+      }
+    }
```

Comment:
I don’t think this is true in general. For example, think about `return !myBool`. We need the exclamation mark here. 

I think we should just omit the space if the current token kind is in a hardcoded list. For now AFAICT that list should contain `as` and `init`.